src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -84,9 +85,8 @@ export default class CancerStudyTreeData
 				cancerTypeId: name
 			});
 		}
-		//cancerTypes = [virtualCohortsCategory].concat(this.priorityCategories).concat(this.rootCancerType, cancerTypes);
-		//studies = virtualCohortStudies.concat(studies);
-		cancerTypes = this.priorityCategories.concat(this.rootCancerType, cancerTypes);
+		cancerTypes = [virtualCohortsCategory].concat(this.priorityCategories).concat(this.rootCancerType, cancerTypes);
+		studies = CancerStudyTreeData.sortNodes(virtualCohortStudies).concat(studies);

```

Comment:
Is it right at we sort the nodes first and THEN concat?  Do we want to sort after concatenation?

Reply:
I think it doesn't matter. Will clean it up.